src/bin/bat/clap_app.rs

```diff
@@ -426,7 +434,9 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      * auto: same as 'full', unless the output is piped.\n  \
                      * plain: disables all available components.\n  \
                      * changes: show Git modification markers.\n  \
-                     * header: show filenames before the content.\n  \
+                     * header: show filenames before the content.\n \
+                     * header-filename: displays the file name.\n  \
+                     * header-filesize: displays the size of the file in human-readable format.\n  \
```

Comment:
What do you think about reusing previous description in the new one for filename, refer to the new one in the old one, and make header-filesize description more similar in style to header-filesize? Like this:

```suggestion
                     * header: legacy alias for 'header-filename'.\n  \
                     * header-filename: show filenames before the content.\n  \
                     * header-filesize: show file sizes before the content.\n  \
```